programs/squads_multisig_program/src/instructions/proposal_vote.rs

```diff
@@ -33,6 +33,33 @@ pub struct ProposalVote<'info> {
     pub proposal: Account<'info, Proposal>,
 }
 
+#[derive(Accounts)]
+pub struct ProposalVoteV2<'info> {
```

Comment:
@vovacodes @bguillaumat need new accounts here for system program. Any thoughts here? should this be specific to cancel?